WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -214,6 +217,11 @@ private async Task StepConnectionConfirmationPhaseAsync(CancellationToken cancel
 						var offendingAliceCounter = 0;
 						await foreach (var offendingAlices in CheckTxoSpendStatusAsync(round, cancel).ConfigureAwait(false))
 						{
+							if (cancel.IsCancellationRequested)
+							{
+								break;
+							}
+
```

Comment:
Not sure if this is useful, I have a hard time to understand the inputs banned incorrectly for `DoubleSpent`.

Reply:
I preferred to remove this change and stick with the concept of this PR for `DoubleSpent` as well. So I'm checking the offenders count and don't ban if too many.